package.json

```diff
@@ -15,8 +15,8 @@
     },
     "license": "SEE LICENSE IN LICENSE",
     "scripts": {
-        "dev": "http-server ./ -c-1 --cors",
-        "server": "http-server ./ --cors",
+        "dev": "node tests/server.mjs",
+        "server": "node tests/server.mjs",
```

Comment:
do we need to keep both?

Reply:
(I would keep just "dev")

Reply:
ah yeah, wanted to ask... I'd just keep `server` as an alias. but no strong opinion here.